common-rest/src/test/java/org/uniprot/api/rest/validation/QueryFieldValidatorTest.java

```diff
@@ -112,7 +112,7 @@ void isValidBooleanSubQueryReturnTrue() {
                 validator.isValid(
```

Comment:
there is a reference of a ccev_* in src/test/resources/uniprot/search-fields.json.